mypy/plugins/dataclasses.py

```diff
@@ -86,7 +86,7 @@
     field_specifiers=("dataclasses.Field", "dataclasses.field"),
 )
 _INTERNAL_REPLACE_SYM_NAME: Final = "__mypy-replace"
-_INTERNAL_POST_INIT_SYM_NAME: Final = "__mypy-__post_init__"
+_INTERNAL_POST_INIT_SYM_NAME: Final = "__mypy-post_init"
```

Comment:
@sobolevn btw, `__mypy-replace` was intentionally class-private (starts with `__` but doesn't end with `__`), aligning the post_init sym to be the same